mobilecoind-dev-faucet/src/slam/mod.rs

```diff
@@ -40,7 +40,7 @@ pub struct SlamParams {
 impl Default for SlamParams {
     fn default() -> Self {
         Self {
-            target_num_tx: 10000,
+            target_num_tx: 100_000,
```

Comment:
yeah, rust allows you to add `_` to numeric literals to try to make them more readable, i usually do that when there's more than 4 zeroes

Reply:
FWIW, most languages allow this: python > 3.6, Java > 7, C# > 7, Haskell, Ruby, JavaScript, C++14 (w/ an apostrophe, because the gods are petty and jealous).

Reply:
Cool TIL :+1: